src/main/java/racingcar/service/RacingcarService.java

```diff
@@ -3,42 +3,43 @@
 import java.util.ArrayList;
 import java.util.List;
 import racingcar.domain.Car;
-import racingcar.dto.CarPerformance;
+import racingcar.domain.Cars;
 import racingcar.dto.RoundResult;
 import racingcar.util.RandomNumberGenerator;
 
 public class RacingcarService {
 
-    private static final int MIN_OIL_AMOUNT = 0;
-    private static final int MAX_OIL_AMOUNT = 9;
+    private static final int MIN_NUMBER_OF_POWER = 0;
+    private static final int MAX_NUMBER_OF_POWER = 9;
 
-    public RoundResult processRound(List<Car> cars) {
-        List<CarPerformance> carPerformances = new ArrayList<>();
-        for (Car car : cars) {
-            car.goIfOilEnough(decideOilAmount());
-            CarPerformance carPerformance = car.getCurrentPerformance();
-            carPerformances.add(carPerformance);
-        }
-        return new RoundResult(carPerformances);
+    public Cars createCars(List<String> carNames) {
+        List<Car> cars = carNames.stream()
+                .map(Car::new)
+                .toList();
+        return new Cars(cars);
```

Comment:
다음 처럼도 가능합니다.
```suggestion
    return carNames.stream()
        .map(Car::new)
        .collect(Collectors.collectingAndThen(Collectors.toList(), Cars::new));
```

static import를 적용하면 조금 더 줄일수 있고요
```
        .collect(collectingAndThen(toList(), Cars::new));
```